faraday/proxy.py

```diff
@@ -19,21 +19,218 @@
 import sys
 import threading
 import time
+import argparse
+import shutil
 
 from flask import Flask
 from flask import request
 
 from faraday.uart import layer_4_service
 
 # Start logging after importing modules
-filename = os.path.abspath("loggingConfig.ini")
-logging.config.fileConfig(filename)
+
+relpath1 = os.path.join('etc', 'faraday')
+relpath2 = os.path.join('..', 'etc', 'faraday')
+setuppath = os.path.join(sys.prefix, 'etc', 'faraday')
+userpath = os.path.join(os.path.expanduser('~'), '.faraday')
+path = ''
+
+for location in os.curdir, relpath1, relpath2, setuppath, userpath:
+    try:
+        logging.config.fileConfig(os.path.join(location, "loggingConfig.ini"))
+        path = location
+        break
+    except ConfigParser.NoSectionError:
+        pass
+
 logger = logging.getLogger('Proxy')
+logger.debug("PATH: " + path)
+
+#Create Proxy configuration file path
+proxyConfigPath = os.path.join(path, "proxy.ini")
+logger.debug('Proxy.ini PATH: ', proxyConfigPath)
```

Comment:
Woops, this is wrong. I need to use a `+` instead of comma to join the two items inside the logging.debug() code. Also, this is redundant and I should delete line 47